modules/generic_worker/manifests/init.pp

```diff
@@ -54,6 +54,10 @@
             # The reboot command in OSX not have --force option
             $reboot_command = '/usr/bin/sudo /sbin/reboot'
 
+            # Since python 3.7.1 require openssl 1.0.2 or greater, and on OSX workers we have openssl 0.98
+            # and python3 is pinned I'll use python3.6 instead python3
+            $python3 = '/usr/local/bin/python3.6'
```

Comment:
:crying_cat_face:

Reply:
Long-term, it may be better to move the quarantine-worker to be a statically compiled binary or to run with an interpreter that is available everywhere (or maybe python code that is backwards-compatible).